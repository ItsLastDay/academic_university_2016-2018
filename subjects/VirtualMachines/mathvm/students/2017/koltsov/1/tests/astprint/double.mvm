double x;
x += 8.0;
