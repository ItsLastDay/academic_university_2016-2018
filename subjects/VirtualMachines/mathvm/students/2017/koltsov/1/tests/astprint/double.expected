double x;
x += 8.000000;
